### ISSUE TITLE
Model: gpt-3.5-turbo

### ISSUE THREAD

**Issue by countertops** [hackdaddy8000/unsuperior-ai-waifu#16]
Hi Bruce,

OpenAI announces an API for ChatGPT.
Can we use Model: gpt-3.5-turbo right now?
How could this be possible with code modified? I am new to coding.

**Comment by hackdaddy8000** [comment]
![image](https://user-images.githubusercontent.com/121913194/222635695-bccb20ee-8308-4903-9d86-0d129cc397bc.png)

I highly doubt it. It's going to take some top tier prompt engineering which I'm too lazy to do.
I believe 3.5 is fine tuned to sound like a nerd so I doubt it would make a good conversational LLM for waifus anyways.

**Comment by countertops** [comment]
Duly noted with thanks! Thanks for your good work, Bruce. Keep it up. I used your work everyday.

**Comment by Yardanico** [comment]
> ![image](https://user-images.githubusercontent.com/121913194/222635695-bccb20ee-8308-4903-9d86-0d129cc397bc.png)
> 
> I highly doubt it. It's going to take some top tier prompt engineering which I'm too lazy to do. I believe 3.5 is fine tuned to sound like a nerd so I doubt it would make a good conversational LLM for waifus anyways.

not really, you don't need a big prompt for it, check out projects like TavernAI, there are some mods with turbo already, and it seems like you can roleplay with AIs easily. Yes, it is soft-filtered, but not that strongly.

And the response times + pricing make up for it
